AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/MainActivity.java

```diff
@@ -69,17 +79,76 @@ public void onSdkInitialized(final AppLovinSdkConfiguration config)
         } );
     }
 
+    private void checkSdkKey()
+    {
+        final String sdkKey = AppLovinSdk.getInstance( getApplicationContext() ).getSdkKey();
+        if ( "YOUR_SDK_KEY".equalsIgnoreCase( sdkKey ) )
+        {
+            new AlertDialog.Builder( this )
```

Comment:
We have a utility method called `showOkAlertDialog()` in `TestAppUtils.java`, we can also add `setCancelable( false )` to it

Reply:
@stnaqvi I noticed that checkSdkKey() method is never used. Do you think we should still go ahead and make the above update within checkSdkKey() ?

Reply:
I think it is supposed to be called somewhere within MainActivity, I will look into it.

Reply:
Hmm I don't think we have it in the new demo app but I'm down to add it in similar fashion to what we have [here](https://github.com/AppLovin/Android-SDK-Demo/blob/6408e9d6122b782b161f540267bfcfb9a2735a92/Android-Demo-App/src/main/java/com/applovin/apps/demoapp/MainActivity.java#L56).